etc/ci.sh

```diff
@@ -78,6 +72,52 @@ cd profiling
 make V=1
 cd ..
 
+if [[ "${TEST_SUITE}" == testpackages ]]
+then
+    # skip carat because building it leads to too much output which floods the log
+    rm -rf carat*
+    # skip linboxing because it hasn't compiled for years
+    rm -rf linboxing*
+    # skip pargap: no MPI present (though we could fix that), and it currently does not
+    # build with the GAP master branch
+    rm -rf pargap*
+    # skip PolymakeInterface: no polynmake installed (TODO: is there a polymake package we can use)
+    rm -rf PolymakeInterface*
+    # HACK to work out timestamp issues with anupq
+    touch anupq*/configure* anupq*/Makefile* anupq*/aclocal.m4
+    # HACK to prevent float from complaining about missing C-XSC (not available in Ubuntu)
+    # and fplll (float 0.7.5 is not compatible with fplll 4.0)
+    perl -pi -e 's;CXSC=yes;CXSC=no;' float*/configure
+    perl -pi -e 's;CXSC=extern;CXSC=no;' float*/configure
+    perl -pi -e 's;FPLLL=yes;FPLLL=no;' float*/configure
+    perl -pi -e 's;FPLLL=extern;FPLLL=no;' float*/configure
+    # PATCH Browse (proper fix has been submitted to authors)
```

Comment:
After GAP 4.8.8 will be out, the Browse patch will not be needed.

Reply:
GAP 4.8.8 is out - I suggest to rebase, remove Browse patch, check that Travis tests pass and merge this.